exporters/variables-scss/src/helpers/colorHelper.ts

```diff
@@ -42,3 +43,31 @@ export const normalizeColor = (hexCode: string): string => {
 
   return `#${processedHex}`;
 };
+
+// Find all hex colors in string and normalize them
```

Comment:
Are the comments needed? Shouldn't the function name be named `findAllHexColorsInStringAndNormalize`? I am ok with it :-)

Just the comment suggests that the name of the function is not enough to understand what the function does.

Reply:
yes, i use alot of comments during the development when i moving things around, we dont need them in the code. i'll remove them